libtests/qtest/json_parse.test

```diff
@@ -120,15 +120,15 @@ my @bad = (
     "stray low surrogate",      # 37
     "high high surrogate",      # 38
     "dangling high surrogate",  # 39
-    "duplicate dictionary key", # 40
+    # 40 "bs char in string"
     "decimal point after minus",# 41
     "e after minus",            # 42
     "missing digit after e",    # 43
     "missing digit after e+/-", # 44
     "tab char in string",       # 45
     "cr char in string",        # 46
     "lf char in string",        # 47
-    "bs char in string",        # 48
+    "bs char in string",        # 40
```

Comment:
These have to be in order. Otherwise, the labels in the test output won't match up with the actual test file.

Reply:
I realized that, but managed to comment the wrong entry out. (I think its worth keeping a comment in the original place as the test logically belongs there)